docs/orleans/host/monitoring/index.md

```diff
@@ -1,14 +1,158 @@
 ---
-title: Runtime monitoring and logs
-description: Explore the various runtime monitoring and logs values in .NET Orleans.
-ms.date: 03/16/2022
+title: Orleans observability
+description: Explore the various runtime monitoring, logging, distributed tracing, and metrics options available in .NET Orleans.
+ms.date: 02/22/2023
+zone_pivot_groups: orleans-version
 ---
 
-# Orleans logs
+# Orleans observability
 
-Orleans leverages [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) for all silo and client logs. For more information, see [Logging in .NET](../../../core/extensions/logging.md).
+One of the most important aspects of a distributed system is observability. Observability is the ability to understand the state of the system at any given time. There are various ways to achieve this, including logging, metrics, and distributed tracing.
 
-## Runtime monitoring
+## Logging
+
+Orleans leverages [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) for all silo and client logs. This means that you can use any logging provider that is compatible with `Microsoft.Extensions.Logging`. Your app code would rely on [dependency injection](../../../core/extensions/dependency-injection.md) to get an instance of <xref:Microsoft.Extensions.Logging.ILogger%601> and use it to log messages. For more information, see [Logging in .NET](../../../core/extensions/logging.md).
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+## Metrics
+
+Metrics are numerical measurements reported over time, most often used to monitor the health of an application and generate alerts. For more information, see [Metrics in .NET](../../../core/diagnostics/metrics.md). Orleans uses the [System.Diagnostics.Metrics](../../../core/diagnostics/compare-metric-apis.md#systemdiagnosticsmetrics) APIs to collect metrics. The metrics are exposed to the [OpenTelemetry](https://opentelemetry.io) project, which exports the metrics to various monitoring systems.
+
+To monitor your app without making any code changes at all, you can use the `dotnet counters` .NET diagnostic tool. To monitor Orleans <xref:System.Diagnostics.ActivitySource> counters, given your desired `{ProcessName}` to monitor, use the `dotnet counters monitor` command as shown:
```

Comment:
```suggestion
To monitor your app without making any code changes at all, you can use the `dotnet counters` .NET diagnostic tool. To monitor Orleans <xref:System.Diagnostics.ActivitySource> counters, given your desired `<ProcessName>` to monitor, use the `dotnet counters monitor` command as shown:
```

Reply:
Angle bracket guidance is from https://review.learn.microsoft.com/en-us/help/contribute/text-formatting-guidelines?branch=main#placeholders.